### DISCUSSION TITLE
xd-AntiSpy - 4.12.5 (GothamX)

### DISCUSSION THREAD

**Discussion by Belim** [builtbybel/xd-AntiSpy#67] [Announcements]
<blockquote class="twitter-tweet"><p lang="en" dir="ltr">Just released a private project: xD-AntiSpy 😎! It&#39;s a fork of the original, picking up where the last version left off. ✨ <a href="https://t.co/AfyGzHEtFw">https://t.co/AfyGzHEtFw</a><a href="https://twitter.com/hashtag/xDAntiSpy?src=hash&amp;ref_src=twsrc%5Etfw">#xDAntiSpy</a> <a href="https://twitter.com/hashtag/xpantispy?src=hash&amp;ref_src=twsrc%5Etfw">#xpantispy</a> <a href="https://twitter.com/hashtag/windows11?src=hash&amp;ref_src=twsrc%5Etfw">#windows11</a> <a href="https://twitter.com/hashtag/windows?src=hash&amp;ref_src=twsrc%5Etfw">#windows</a> <a href="https://twitter.com/hashtag/app?src=hash&amp;ref_src=twsrc%5Etfw">#app</a> <a href="https://twitter.com/hashtag/microsoft?src=hash&amp;ref_src=twsrc%5Etfw">#microsoft</a> <a href="https://twitter.com/hashtag/privacy?src=hash&amp;ref_src=twsrc%5Etfw">#privacy</a> <a href="https://t.co/wajGZtGLn5">pic.twitter.com/wajGZtGLn5</a></p>&mdash; Belim (@builtbybel) <a href="https://twitter.com/builtbybel/status/1805906464843694431?ref_src=twsrc%5Etfw">June 26, 2024</a></blockquote>

- Operating under the codename **GothamX**, I’ve **introduced a new dynamic settings form**, allowing centralized control over all xd-AntiSpy pages. The settings page, or cPanel, sports a classic old-school look because it was migrated from another legacy cpp project. It may not have the modern aesthetics of the Microsoft Settings app, but it offers a level of speed and reliability that the Microsoft app could often only dream of 🥲 
- **A Plugins/Marketplace page is now live**, dynamically pulling all available plugins from GitHub and offering them for direct download. Some plugins, like the Debloater, come pre-installed, while others that extend settings in the main form can now be accessed directly through the new Plugins page. This keeps the core xd-AntiSpy bundle lean and clean! Check back regularly to discover new plugins and extensions. Keep in mind that GitHub API rate limits can be hit..so if that happens, patience is key!
**As a test, I’ve also included Joe Thio's "[Super God Mode]()"  script as a plugin - check out the presentation video for more details**
- Load times, especially at app startup, have been significantly improved

https://github.com/user-attachments/assets/d9af3003-3331-4c49-a3d5-e08e81931d2f

<hr /><em>This discussion was created from the release <a href='https://github.com/builtbybel/xd-AntiSpy/releases/tag/4.12.5'>xd-AntiSpy - 4.12.5 (GothamX)</a>.</em>

**Comment by alexifeu** [comment]
so you always make new apps that do the same thing over and over again but with a different name lol? You chnaged the name so often lol.
